futures-rs-test-all/Cargo.toml

```diff
@@ -14,7 +14,7 @@ composability, and iterator-like interfaces.
 """
 
 [dependencies]
-log = "0.3"
+log = "= 0.3.6"
```

Comment:
I think the way this is usually written is `log = "=0.3.6"` (note the lack of space after `=`), but I may be wrong.